app/src/main/java/io/pumpkinz/pumpkinreader/model/Item.java

```diff
@@ -0,0 +1,129 @@
+package io.pumpkinz.pumpkinreader.model;
+
+import java.util.Date;
+
+
+public abstract class Item {
```

Comment:
We can't use Item as a return value for Retrofit container because we will only get `id, deleted, type, by, time, text, dead` (Item's fields) and lose the others (`url, title`, etc).

We can't use its child class (e.g. `Story, Job, etc`) either as we don't know the type yet.